app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -60,9 +84,14 @@ def perform_vmware_to_ovirt_import(params)
           :sparse         => params[:sparse],
         )
       )
+      ems_ref_for_vm_id(import.vm.id)
     end
   end
 
+  def ems_ref_for_vm_id(vm_id)
+    "/api/vms/#{vm_id}"
```

Comment:
@matobet I think you should get it from import.vm.href but that one has /ovirt-engine/ in the beginning so it should be removed. I think it is more stable to remove the prefix than to build the entire ref in case of future changes. 
@jhernand waht is the best approach in this case?